src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -720,11 +752,18 @@ public void HandleDatabaseRecordChange(DatabaseTopology topology)
                         continue;
                     }
 
-                    var whoseTaskIsIt = OngoingTasksUtils.WhoseTaskIsIt(_serverStore, topology, subscriptionState, subscriptionState, _db.NotificationCenter);
+                    if (_serverStore.Engine.CurrentState == RachisState.Passive)
```

Comment:
this can happen? in order to create a database you need to ensure that you are not passive, right?

Reply:
As far as I understood, yes. When Node is removed from cluster